<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportControlWidget</class>
 <widget class="QWidget" name="ReportControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>440</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="gbReportStorage">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>5</y>
     <width>481</width>
     <height>421</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Report Storage</string>
   </property>
   <widget class="QCheckBox" name="cbSendViaEmail">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Report via Email</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbSaveToFile">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Report to File</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDeleteOlderThan">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Reports Older Than</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbDeleteOlderThan">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>80</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <property name="maxVisibleItems">
     <number>4</number>
    </property>
    <item>
     <property name="text">
      <string>1 Day</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1 Week</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2 Weeks</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1 Month</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="cbStoreToUsbDrive">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Store to USB Drive</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbDownloadSelectedReports">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>260</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbDeleteSelectedReports">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>210</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QListView" name="lvAvailableReports">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>351</width>
      <height>211</height>
     </rect>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblAvailableReports">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Available Reports</string>
    </property>
   </widget>
   <widget class="Line" name="lnSeparator">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>461</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="cbUsbDrives">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbSelectAllReports">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>310</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Select All</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbClearAllReports">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>360</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear All</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbImmediateReport">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>190</y>
     <width>348</width>
     <height>236</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Immediate Report</string>
   </property>
   <widget class="QDateTimeEdit" name="dtReportStart">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>40</y>
      <width>207</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumDateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2016</year>
      <month>6</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="displayFormat">
     <string>yyyy-MM-dd HH:mm:ss</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dtReportEnd">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>100</y>
      <width>207</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumDateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2016</year>
      <month>6</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="displayFormat">
     <string>yyyy-MM-dd HH:mm:ss</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblReportStart">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>40</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Date/Time</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblReportEnd">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>100</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>End Date/Time</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pbGenerateReport">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Report</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbAutoReport">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>348</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Enable Auto Reporting</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="lblStartDateTime">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>100</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Date/Time</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblFrequency">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>40</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Frequency</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbFrequency">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>40</y>
      <width>207</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Each Activity</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hourly</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Every 12 Hours</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Daily</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Weekly</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Monthly</string>
     </property>
    </item>
   </widget>
   <widget class="QDateTimeEdit" name="dtStartDateTime">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>100</y>
      <width>207</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumDateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2016</year>
      <month>6</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="displayFormat">
     <string>yyyy-MM-dd HH:mm:ss</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
